src/Test/Perf/infra/automation.csx

```diff
@@ -13,5 +14,8 @@ ShellOutVital(Path.Combine(RoslynDirectory(), "Restore.cmd"), "", workingDirecto
 // Build Roslyn in Release Mode
 ShellOutVital("msbuild", "./Roslyn.sln /p:Configuration=Release", workingDirectory: RoslynDirectory());
 
+// Run DownloadTools before using the TraceManager because TraceManager uses the downloaded CPC binaries 
+DownloadTools();
```

Comment:
Like we discussed offline, bootstrap.bat will be run by everyone so we dont want to make it a part of BootStrap instead I will create `internal_BootStrap`(or a better name) which the internal people will have to run apart from `BootStrap`